packages/cli/generators/model/index.js

```diff
@@ -260,10 +223,26 @@ module.exports = class ModelGenerator extends ArtifactGenerator {
       if (NON_TS_TYPES.includes(val.tsType)) {
         val.tsType = 'string';
       }
+
+      if (
+        val.defaultValue &&
+        NON_TS_TYPES.concat(['string', 'any']).includes(val.type)
+      ) {
+        val.defaultValue = `'${val.defaultValue}'`;
+      }
+
+      // Convert Type to include '' for template
+      val.type = `'${val.type}'`;
+
+      if (!val.required) {
+        delete val.required;
```

Comment:
Is this necessary? What I'm getting from this is if `val.required` does not exist, delete the key off of `val`. Is the code manually defining the key as `undefined` or `null` somewhere?